src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
@mcmachado Hum, I am not quite sure. First, wait uses the emulate function under the hood which is a private function of the StellaEnvironment class. Second, the wait function could be useful for another game. And last, I cannot see why this is an issue as it does not break the API or anything but just adds a new feature. Could you explain ?